website/source/docs/providers/aws/r/elasticache_replication_group.html.markdown

```diff
@@ -0,0 +1,77 @@
+---
+layout: "aws"
+page_title: "AWS: aws_elasticache_replication_group"
+sidebar_current: "docs-aws-resource-elasticache-replication-group"
+description: |-
+  Provides an ElastiCache Replication Group resource.
+---
+
+# aws\_elasticache\_replication\_group
+
+Provides an ElastiCache Replication Group resource.
+
+## Example Usage
+
+```
+resource "aws_elasticache_replication_group" "redis" {
+    replication_group_id = "users-redis"
+    description = "users redis"
+    engine = "redis"
+    cache_node_type = "cache.m3.medium"
+    num_cache_clusters = 2
+    automatic_failover = true
+    subnet_group_name = "${aws_elasticache_subnet_group.redis.name}"
+    security_group_ids = ["${aws_security_group.redis.id}"]
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `replication_group_id` – (Required) Replication group identifier. This
+parameter is stored as a lowercase string
+
+* `description` – (Required) The description of the replication group.
+
+* `engine` – (Optional) The name of the cache engine to be used for the cache clusters in this replication group.
+ The only current valid value is `redis`
```

Comment:
I'd reword the last sentence to "The only value that is currently valid is `redis`"